app/helpers/avatar_helper.rb

```diff
@@ -113,12 +113,20 @@ def gravatar_default_options
         :secure => true,
         :default => default_avatar_url
       },
-      :class => 'user-avatar'
+      :class => 'avatar left mrm'
     }
   end
   
-  def user_avatar_from_api(user, options = {size: 24})
-    image_tag(user_avatar_url_from_api(user, options), width: options[:size], height: options[:size], class: 'user-avatar') if user_avatar_url_from_api(user)
+  def user_avatar_from_api(user, options = {})
+    options = user_avatar_from_api_default_options.deep_merge(options)
+    image_tag(user_avatar_url_from_api(user, options), width: options[:size], height: options[:size], class: options[:class]) if user_avatar_url_from_api(user)
+  end
+  
+  def user_avatar_from_api_default_options
+    {
+      :class => 'avatar left mrm',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.